### ISSUE TITLE
Failed to load everything with neovim commit 43956de

### ISSUE THREAD

**Issue by shinymimiq** [ChristianChiarulli/LunarVim#441]
In neovim built with the commit #https://github.com/neovim/neovim/commit/43956dea55a49a847a0f2e5d9e0f48384cb34635
LunarVim failed to load everything, it raised such error:

```
E5113: Error while calling lua chunk: ...ovim/HEAD-43956de_2/share/nvim/runtime/lua/vim/_meta.lua:21: attempt to index local
'info' (a nil value)
```

Not quite sure if regressions introduced by this commit or something bad happened on LunarVim side.

**Comment by shinymimiq** [comment]
For reference: 
Good version:
```
❯ nv --version
NVIM v0.5.0-dev+1358-g192ea01ed
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/Users/shinymimiq/Developer/neovim/build/config -I/Users/shinymimiq/Developer/neovim/src -I/Users/shinymimiq/Developer/neovim/.deps/usr/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk/usr/include -I/opt/homebrew/opt/gettext/include -I/Users/shinymimiq/Developer/neovim/build/src/nvim/auto -I/Users/shinymimiq/Developer/neovim/build/include
Compiled by shinymimiq@Shinys-MacBook-Air.local

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"

Run :checkhealth for more info

```

neovim version with problem
```
 ❯ /opt/homebrew/Cellar/neovim/HEAD-43956de_2/bin/nvim --version
NVIM v0.5.0-dev+43956dea5
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: clang -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/tmp/neovim-20210528-55091-12px3r/build/config -I/tmp/neovim-20210528-55091-12px3r/src -I/opt/homebrew/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include -I/opt/homebrew/opt/gettext/include -I/tmp/neovim-20210528-55091-12px3r/build/src/nvim/auto -I/tmp/neovim-20210528-55091-12px3r/build/include
Compiled by shinymimiq@Shinys-MacBook-Air.local

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/opt/homebrew/Cellar/neovim/HEAD-43956de_2/share/nvim"

Run :checkhealth for more info
```

**Comment by ChristianChiarulli** [comment]
I think this should be fixed

**Comment by Antonio-Bennett** [comment]
@ChristianChiarulli Hey Christian turns out vim.o._t_Co needs to be vim.go.t_Co now fixed it for me
https://github.com/neovim/neovim/issues/14662#issuecomment-850797636

**Comment by clason** [comment]
Note that `t_Co` and friends aren't supported by neovim anymore and have been deprecated for a long time, so that option shouldn't have been set in the first place ;)

**Comment by shinymimiq** [comment]
Resolved by commit [`4ddcaf5`](https://github.com/ChristianChiarulli/LunarVim/commit/4ddcaf55a9047807c211d1fb31fc86dff7971162)
